README.md

```diff
@@ -1 +1,70 @@
-# Leaderboard
\ No newline at end of file
+
+
+# Leaderboard
+
+> Leaderboard list app
+
+**Additional description about the project and its features.**
+- This is a leaderboard list app that consumes an API that implements the actions behind the refresh and submit.
+
+## Linter for html&css (Stylelint)
+-https://github.com/microverseinc/linters-config/tree/master/html-css
+
+## linter for html&css + javascript (ESlint)
+-https://github.com/microverseinc/linters-config/tree/master/html-css-js
+
+## ⚒️ Built With
+
+- WEBPACK & JAVASCRIPT & CSS
+
+## Run Project
+
+- `npm install`
+- `npm run start # this will make webpack watching for your changes in code`
+
+## Open in browser
+
+- open dist/index.html
+
+## Live Demo
+[not availbale for now]
+
+## 🧰 Tools
+
+The following tools help make it easier to run this code.
+
+- [Visual Studio Code](https://code.visualstudio.com/): A source code editor
+
+## 🔥 Getting Started
+
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+[Download the ZIP from this location](https://github.com/altontonn/Leaderboard/archive/refs/heads/main.zip), or run the following [git](https://github.com/altontonn/Leaderboard.git) command to clone the files to your machine:
+
+```bash
+git https://github.com/altontonn/library.git
+```
+
+- Once the files are downloaded or cloned on your machine, open the "Portfolio" folder in [Visual Studio Code(VSCode)](https://code.visualstudio.com/)
+- Optionally you can simply open the index.html file located in the "personal-portfolio" folder with your preferred browser
+
+## 🙎🏾‍♂️ Authors
+
+- GitHub: [Newton Alumasa](https://github.com/altontonn)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/altontonn/Leaderboard/issues).
+
+## Acknowledgements
+
+- Microverse
+
+## 👊🏾 Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link to your license is broken. Please don't forget to add a license to your project and I recommend using the [MIT license](https://choosealicense.com/licenses/mit/), Set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). another resource is [any other license](https://choosealicense.com/licenses/). :smile: